### PR TITLE
show if tags are installed

### PR DESCRIPTION
**Pull request by jakobroehrl**
Signed-off-by: Jakob Röhrl <jakob.roehrl@web.de>
Solves: #499

### CODE REVIEW

File: src/services/AreTagsInstalled.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/**
+ * @copyright Copyright (c) 2018 John Molakvoæ <skjnldsv@protonmail.com>
+ *
+ * @author John Molakvoæ <skjnldsv@protonmail.com>
```
Comment (skjnldsv) [inline src/services/AreTagsInstalled.js:4]: It's your code :)

**Comment by skjnldsv** [comment]
You need to ship the production builds :)
Capture d’écran_2020-10-20_15-00-29

**Comment by jakobroehrl** [comment]
> 
> 
> You need to ship the production builds :)
> Capture d’écran_2020-10-20_15-00-29

OK, how to do this?

**Comment by skjnldsv** [comment]
`npm run build` instead of `npm run dev/watch`

**Comment by skjnldsv** [comment]
Or use the compile bot like this (`/compile /js`, `/compile amend /js`)
/compile /js

**Review by Mikescops — CHANGES_REQUESTED** [review]
PR is good except that the path `apps/photos/tags` is still accessible when the app tags is disabled. Can you redirect to `/` when it's the case?

**Comment by Mikescops** [comment]
Closes #499

**Comment by jakobroehrl** [comment]
> 
> 
> PR is good except that the path `apps/photos/tags` is still accessible when the app tags is disabled. Can you redirect to `/` when it's the case?

Good idea. I now have the areTagsInstalled value here: https://github.com/nextcloud/photos/blob/master/src/router/index.js
How to adapt the routes?

**Comment by Mikescops** [comment]
@jakobroehrl you can do something similar: isMapsInstalled

**Comment by jakobroehrl** [comment]
> 
> 
> @jakobroehrl you can do something similar: isMapsInstalled

Tried these solutions, doesn't work:
https://github.com/nextcloud/photos/blob/a706e9c0bcecc07a5d3c035c2cb1b6265739a986/src/router/index.js
https://github.com/nextcloud/photos/blob/bfb72e18a9643b9150ae74f0e8c8fc6b0802a24b/src/router/index.js

What can I do?

File: src/router/index.js [hunk 2]
```diff
@@ -97,7 +103,8 @@ export default new Router({
 			}),
 		},
 		{
-			path: '/tags/:path*',
+			// path: '/tags/:path*',
+			path: tagsPath,
```
Comment (Mikescops) [inline src/router/index.js:107]: The router path should stay `'/tags/:path*'`

Comment (Mikescops) [reply src/router/index.js:107]: you can add 
```
beforeEnter() {
                        if (!areTagsInstalled) {
				window.open(generateUrl('/apps/photos'))
			},
},
```

**Comment by Mikescops** [comment]
@jakobroehrl I added a comment to help you

File: src/router/index.js (hunk 2 above)
Comment (jakobroehrl) [reply src/router/index.js:107]: With this change the tags route isn't working any more

Comment (jakobroehrl) [reply src/router/index.js:107]: @Mikescops
any ideas what I can do? beforeEnter code is unfortunately not working.

Comment (Mikescops) [reply src/router/index.js:107]: let me checkout you're code locally

**Comment by Mikescops** [comment]
@jakobroehrl here you are

**Comment by jakobroehrl** [comment]
> 
> 
> @jakobroehrl here you are

Great, thank you. Now let's merge it in.

**Review by Mikescops — APPROVED** [review]

**Pull request merged**